### ISSUE TITLE
Webform - Allowed File Types

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#353]
Follow-up from #201

### Problem
Form builders may have the ability with file uploads to specify additional file types.

### Proposed Solution
- Decide whether or not we are okay with the file types allowed through webform configuration (admin/structure/webform/config/elements)
- Confirm that non-admins cannot specify additional file types on a per component basis.
- If non-admins can, look for an issue related to this on the webform project issue queue.

**Comment by joewhitsitt** [comment]
Besides deciding on which file types we want to allow, this is a good first issue.

**Comment by ethanstjohn** [comment]
![Screen Shot 2019-07-31 at 11 03 35 AM](https://user-images.githubusercontent.com/42824212/62228134-b8db2200-b382-11e9-98b1-592cf4fbf7ba.png)

**Comment by ethanstjohn** [comment]
While masqueraded as a webmaster I was able to add "md" to the allowed file extensions of a test webform I created.

**Comment by joewhitsitt** [comment]
If this is the case and not seeing any issues specifically for this, we need to disable these fields.

As for default allowed, I say we just allow the ones we currently do for D7 Standard Service. We can revisit them later or as people complain.

**Comment by ethanstjohn** [comment]
I have only found 1 issue regarding this and it was from 2 years ago and nobody answered it.

**Comment by joewhitsitt** [comment]
You may want to dig through our current webforms in D7 regarding disabling the allowed types per component. https://github.com/ITS-UofIowa/sitenow_standard_profile/tree/7.x-3.x/modules/custom/sitenow_forms

**Comment by ethanstjohn** [comment]
I dug through what I could. I couldn't find where to change the default allowed file upload types or file types in general. But I found which default types are allowed.
![Screen Shot 2019-08-06 at 11 33 45 AM](https://user-images.githubusercontent.com/42824212/62558148-15848400-b83e-11e9-9667-2e565fe3fc30.png)
![Screen Shot 2019-08-06 at 11 33 10 AM](https://user-images.githubusercontent.com/42824212/62558169-1d442880-b83e-11e9-87e7-f1f4c1b0cc9e.png)

**Comment by joewhitsitt** [comment]
Okay, I just looked at this. Except for a few new ones, the out of the box types closely matches the D7 version. Additions include .svg .odf (open office document) .odp (open office presentation) .webm (open video). I think I am okay with this. Additionally, I don't see the ability to add others. A webmaster can add another file type to the allowed field, but it is stripped on save and I was unable to upload a .md file.

I think we are good. No action needed. Closing.
